JSDemos/Demos/TextBox/Overview/description.md

```diff
@@ -1 +1,34 @@
-The **TextBox** is&nbsp;a&nbsp;widget that enables a&nbsp;user to&nbsp;enter and edit a&nbsp;single line of&nbsp;text.
\ No newline at end of file
+The [TextBox](/Documentation/ApiReference/UI_Widgets/dxTextBox/) is a UI component that allows users to enter and edit a single line of text. This demo illustrates the following  **TextBox** properties:
+
+- [value](/Documentation/ApiReference/UI_Widgets/dxTextBox/Configuration/#value)     
+A value the **TextBox** displays.
+
+- [placeholder](/Documentation/ApiReference/UI_Widgets/dxTextBox/Configuration/#placeholder)       
+A text that the **TextBox** displays when the **value** is not defined.
+
+- [showClearButton](/Documentation/ApiReference/UI_Widgets/dxTextBox/Configuration/#showClearButton)        
+Specifies whether to display the button that clears the **TextBox** **value**.
+
+- [mode](/Documentation/ApiReference/UI_Widgets/dxTextBox/Configuration/#mode)        
+Affects a set of keyboard characters displayed on a mobile device when the **TextBox** gets focus and adds specific visual features to the UI component. In this example, the **mode** is set to *"password"* so that entered characters are obscured, and the password cannot be read.
```

Comment:
I'd change "specific features". The word "specific" doesn't add much value, I think. So maybe something like this:

```suggestion
Affects a set of keyboard characters displayed on a mobile device when the **TextBox** gets focus and modifies the UI component's display style. In this example, the **mode** is set to *"password"* so that entered characters are obscured, and the password cannot be read.
```